### PR TITLE
DOCS-11157 multiline logs not currently joined

### PR DESCRIPTION
**Pull request by IsaacEldridge**
This PR changes the note under Forward Logs to a list of current limitations

### CODE REVIEW

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 1]
```diff
@@ -101,15 +101,15 @@ Runtime Fabric on VMs / Bare Metal supports log forwarding to:
 ** Splunk
 ** CloudWatch
 
-[NOTE]
-====
-* Only one external log forwarding configuration is supported at any one time. However, you can enable Anypoint log forwarding and an external log forwarding service at the same time.
+=== Limitations for External Log Forwarding
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-fabric-logs.adoc:104]: @hegderohit89 this is the new section

Comment (hegderohit89) [reply modules/ROOT/pages/runtime-fabric-logs.adoc:104]: This is great, we can extend this section in the future

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 2]
```diff
@@ -101,15 +101,15 @@ Runtime Fabric on VMs / Bare Metal supports log forwarding to:
...
+
+* Only one external log forwarding configuration is supported at any one time. However, you can enable Anypoint Platform log forwarding and an external log forwarding service at the same time.
 * External log forwarding does not currently support proxies that are configured for a Runtime Fabric cluster. You must explicitly configure firewall rules to allow the outbound IP address and port of the external log forwarding service.
-====
+* External log forwarding does not currently join multiline log entries during log file rotation.
```
Comment (hegderohit89) [inline modules/ROOT/pages/runtime-fabric-logs.adoc:108]: @kmendiratta-crm

**Review by sarjeet2013 — APPROVED** [review]
LGTM

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**